### PR TITLE
Basic pages

### PR DESCRIPTION
**Pull request by Kwolik**
Customize the appearance of the pages (also mobile):

1.  404
2.  PC and PP,
3. Agenda
4. Keynote Speakers
5. About
6. Contact

### CODE REVIEW

File: web-client/src/components/ContactInfo.jsx [hunk 1]
```diff
@@ -35,7 +43,11 @@ export default function ContactInfo(props) {
 
   return (
     <div className={style.main}>
-      <img src={props.path} className={style.photo} alt={props.alternativeText}></img>
+      <img
+        src={props.path}
+        className={style.photo}
+        alt={props.alternativeText}
+      ></img>
```
Comment (JustD4nTe) [inline web-client/src/components/ContactInfo.jsx]:
```suggestion
      />
```

File: web-client/src/pages/error-404.js [hunk 2]
```diff
@@ -1,8 +1,30 @@
...
-export default Error404;
+  return (
+    <div>
+      <h1 className={style.title}>Oops! Page not found</h1>
+      <NavLink exact to="/" className={style.linkButton}>
+        <Button color="primary" variant="contained" className={style.linkButton}>
+          Go to home
+        </Button>
+      </NavLink>
+      <div><img className={style.photo} src={background} alt="404"/></div>
```
Comment (JustD4nTe) [inline web-client/src/pages/error-404.js]:
```suggestion
      <div><img className={style.photo} src={background} alt=""/></div>
```

File: web-client/src/components/Account/AboutProfil.jsx [hunk 3]
```diff
@@ -9,25 +9,44 @@ export default function AboutProfil(props) {
...
+        width: "124px",
+      },
     },
-    h3: {
-      textAlign: "left",
-    },
-    p: {
-      textAlign: "left",
+    specialization: {
+      fontWeight: "600",
```
Comment (upsu90) [inline web-client/src/components/Account/AboutProfil.jsx]:
```suggestion
      fontWeight: "bold",
```

**Review by JustD4nTe — CHANGES_REQUESTED** [review]

File: web-client/src/components/ContactMap.jsx [hunk 4]
```diff
@@ -6,7 +6,12 @@ export default function ContactMap(props) {
   const style = makeStyles({
     map: {
       width: "100%",
-      marginLeft: "40px",
+      marginLeft: "72px",
+      "@media (max-width: 1000px)": {
+        marginLeft: "0px",
```
Comment (upsu90) [inline web-client/src/components/ContactMap.jsx]:
```suggestion
        marginLeft: "0",
```

File: web-client/src/pages/error-404.js (hunk 2 above)
Comment (Kwolik) [reply web-client/src/pages/error-404.js]: must be alt after all

File: web-client/src/components/KeynoteSpeaker.jsx [hunk 5]
```diff
@@ -7,30 +7,57 @@ export default function KeynoteSpeaker(props) {
...
+      marginTop: "48px",
+      "@media (max-width: 768px)": {
+        fontSize: "12px",
+        marginTop: "16px",
+        width: "296px",
+        lineHeight: "16px",
+      },
+    },
+    spec: {
+      fontWeight: "600",
```
Comment (upsu90) [inline web-client/src/components/KeynoteSpeaker.jsx]:
```suggestion
      fontWeight: "bold",
```

**Review by upsu90 — CHANGES_REQUESTED** [review]
please change order of texts and photos (text, photo, text, photo)
image

image
I think we can align it to center

File: web-client/src/pages/error-404.js (hunk 2 above)
Comment (upsu90) [reply web-client/src/pages/error-404.js]: yes but it's better to leave alt without text if this photo is only decorative

**Review by KamilDonda — APPROVED** [review]

**Review by JustD4nTe — APPROVED** [review]

**Review by upsu90 — CHANGES_REQUESTED** [review]
image
image
please fix it

**Review by upsu90 — APPROVED** [review]

**Pull request merged**